src/composables/useConversationInfo.js

```diff
@@ -81,7 +83,11 @@ export function useConversationInfo({
 			return t('spreed', 'Joining conversation …')
 		}
 
-		if (exposeMessages.value === false || !hasLastMessage.value) {
+		if (exposeDescription) {
```

Comment:
Last message should have priority, I believe (if it's exposed). Maybe, put it inside the next check?

```js
		if (exposeMessages === false || !hasLastMessage.value) {
			return exposeDescription ? item.value?.description : ''
		}
```